notebooks/jpldd_linedet_testbed.py

```diff
@@ -0,0 +1,97 @@
+from gluefactory.models import get_model
```

Comment:
Please add a comment at the beginning of this file explaining what this is exactly doing and what dataset/model is used.

Reply:
Done.